EntityFramework.Exceptions.Tests/DatabaseTests.cs

```diff
@@ -100,7 +100,7 @@ public virtual async Task NotHandledViolationReThrowsOriginalException()
     public virtual async Task DeleteParentItemThrowsReferenceConstraintException()
     {
         var product = new Product { Name = "AN" };
-        var productPriceHistory = new ProductPriceHistory { Product = product, Price = 15.27m, EffectiveDate = DateTimeOffset.UtcNow.Date.AddDays(-10) };
+        var productPriceHistory = new ProductPriceHistory { Product = product, Price = 15.27m, EffectiveDate = DateTime.UtcNow.Date.AddDays(-10) };
```

Comment:
Why this change?

Reply:
Sorry, forgot to mention that.
That's because [of this](https://www.npgsql.org/efcore/release-notes/6.0.html#detailed-notes)
This part gave me an exception during test:
```
    Microsoft.EntityFrameworkCore.DbUpdateException : An error occurred while saving the entity changes. See the inner exception for details.
---- System.InvalidCastException : Cannot write DateTimeOffset with Offset=03:00:00 to PostgreSQL type 'timestamp with time zone', only offset 0 (UTC) is supported. Note that it's not possible to mix DateTimes with different Kinds in an array/range. See the Npgsql.EnableLegacyTimestampBehavior AppContext switch to enable legacy behavior.
```
Not sure why it was converted with offset of 03 though `UtcNow` was used but didn't want to waste time digging it since it wasn't relevant in this case.

Reply:
Should I change something in this PR?

Reply:
@uhfath No, but you can [make a donation](https://ko-fi.com/giorgi)